### PR TITLE
Feature/pages

### PR DESCRIPTION
**Pull request by LauraHEVA**

### CODE REVIEW

File: src/components/ListBuzzs/ListBuzzs.tsx [hunk 1]
```diff
@@ -11,12 +11,42 @@ import { ListItem } from "../../styles/globalStyledComponents";
...
+        currentOffset,
+        currentOffset + buzzsPerPage
+      );
+      pages.push(pageBuzzs);
+      currentOffset += buzzsPerPage;
+    }
+  } else {
+    pages.push(arrayAllBuzzs);
+  }
+  console.log(pages);
```
Comment (oleguerBDN) [inline src/components/ListBuzzs/ListBuzzs.tsx:41]: Is this a console.log in production? :(

File: src/pages/HomePage.test.tsx [hunk 2]
```diff
@@ -25,21 +25,21 @@ describe("Given a HomePage component", () => {
...
-    //     </BrowserRouter>
-    //   );
-
-    //   // FIXME MOCKEAR USER IS LOGGED IN
-    //   const buttonNewBuzz = screen.getByRole("button");
-
-    //   expect(buttonNewBuzz).toBeInTheDocument();
-    //   expect(buttonNewBuzz.textContent).toBe("New Buzz");
-    // });
+    test.skip("Then it should render a button with the text 'New Buzz' inside", () => {
```
Comment (oleguerBDN) [inline src/pages/HomePage.test.tsx:28]: Why the skip?

Comment (LauraHEVA) [reply src/pages/HomePage.test.tsx:28]: With the conditional rendering the test is not find the button. It needs to be fixed

File: src/components/ListBuzzs/ListBuzzs.tsx (hunk 1 above)
Comment (LauraHEVA) [reply src/components/ListBuzzs/ListBuzzs.tsx]: It looks that husky didn't work...
Fixed it!!

**Pull request merged**